content/applications/productivity/iot/config/connect.rst

```diff
@@ -28,54 +31,78 @@ Ethernet Connection
 #. Read the Pairing Code from a screen or a receipt printer connected to the IoT Box.
 
    .. image:: connect/connect04.png
+       :align: center
+       :alt: Pairing code for a wired connection.
 
-#. Input the Pairing Code and click on the Pair button.
+..
+   _Image needs to be updated.
+
+#. Input the Pairing Code on the IoT module of the database and click on the Pair button.
 
 .. note::
-   Recent changes in modern web browsers forced us to modify the connection wizard.
-   If your screen is different from the screenshots, make sure that the
-   ``iot_pairing`` module is installed.
+   Recent changes in modern web browsers forced us to modify the connection wizard. If the screen is
```

Comment:
well, it's been a few year that the doc was wrote so "recent" is probably not relevant, maybe we can change the "recent changes..." with "Some changes were made..." or something like this.

Reply:
Noted thanks, I've made this change.